### PR TITLE
Casper crates bump + casper-node fix + event types verification

### PR DESCRIPTION
**Pull request by zie1ony**

### CODE REVIEW

File: odra-casper/test-env/src/env.rs [hunk 1]
```diff
@@ -81,13 +81,16 @@ impl CasperTestEnv {
...
 
-        let chainspec_path =
-            PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("resources/chainspec.toml");
+        // Uncomment for testing.
+        // let chainspec_path =
+        //     PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("resources/chainspec.toml");
+        // let mut builder = InMemoryWasmTestBuilder::new_with_chainspec(chainspec_path, None);
 
-        let mut builder = InMemoryWasmTestBuilder::new_with_chainspec(chainspec_path, None);
+        let mut builder = InMemoryWasmTestBuilder::default();
```
Comment (kpob) [inline odra-casper/test-env/src/env.rs]: Why don't we use a chainspec from a file?

**Review by kpob — APPROVED** [review]

**Review by kpob — COMMENTED** [review]
Update the `casper-contract` version in `proxy-caller`. It would be best to add `casper-contract` at the workspace level.

**Review by kubaplas — APPROVED** [review]

File: .github/workflows/odra-ci.yml
Comment (kpob) [inline .github/workflows/odra-ci.yml:1]: :hankey:

File: justfile
Comment (kpob) [inline justfile:1]: :hankey:

**Review by kpob — APPROVED** [review]

**Pull request merged**